File: Math/553. Optimal Division/553. Optimal Division.java
class Solution {
    public String optimalDivision(int[] nums) {
        if (nums.length == 1) return String.valueOf(nums[0]);
        if (nums.length == 2) return String.valueOf(nums[0])+ "/" + String.valueOf(nums[1]);
        String rets = "";
        rets += String.valueOf(nums[0])+ "/(";
        for (int i = 1; i < nums.length; i++) {
            rets += String.valueOf(nums[i]) + ((i == (nums.length-1)) ?  "" : "/");
        }
        rets += ")";
        return rets;
    }
}

File: Math/553. Optimal Division/553. Optimal Division.cpp
class Solution {
public:
    string optimalDivision(vector<int>& nums) {
        if (nums.size() == 1) return to_string(nums[0]);
        if (nums.size() == 2) return to_string(nums[0])+ "/" + to_string(nums[1]);
        string rets = to_string(nums[0])+ "/(";
        for (int i = 1; i < nums.size(); i++) {
            rets += to_string(nums[i]) + ((i == (nums.size()-1)) ?  "" : "/");
        }
        rets += ")";
        return rets;
    }
};
